pkg/broker/server/sse.go

```diff
@@ -16,15 +17,21 @@ import (
 const patience time.Duration = time.Second * 1
 
 type subscription struct {
-	topic string
-	ch    chan []byte
+	topic      string
+	exactMatch bool
+	ch         chan []byte
 }
 
 type event struct {
 	topic string
 	data  []byte
 }
 
+type clientValue struct {
+	value      int
```

Comment:
We probably don't need this struct type at all.  We only need to store the value in the radix tree as `map[chan []byte]bool` where bool is just the `exactMatch` value.  We don't need this `value` field...